project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/base/_mixins.scss

```diff
@@ -10,7 +10,6 @@
   background-color: $color-background;
   border-top: 1px solid $color-gray-light;
   border-bottom: 1px solid $color-gray-light;
-  padding: 2rem 0;
```

Comment:
@nirgn975 
There are many other places that use this mixin, hence we can't change it, you can create a new mixin and include it over here and then use it wherever you need.

something like:
```SCSS

@mixin grey_bar_base {
  background-color: $color-background;
  border-top: 1px solid $color-gray-light;
  border-bottom: 1px solid $color-gray-light;
}

@mixin grey_bar {
  @include grey_bar_base;
  padding: 2rem 0;
}
```

and then use `grey_bar_base` wherever you need.